localtypings/pxtpackage.d.ts

```diff
@@ -173,28 +173,51 @@ declare namespace pxt {
         questions: SnippetQuestions[];
     }
 
-    type SnippetAnswerTypes = any; // TODO(jb) Should include custom answer types for number, enums, string, image
+    type SnippetAnswerTypes = 'number' | 'text' | 'dropdown' | 'spriteEditor' | string; // TODO(jb) Should include custom answer types for number, enums, string, image
 
-    interface SnippetGoToParameters {
-        [tokenType: string]: number;
+    interface SnippetGoToOptions {
+        question?: number;
```

Comment:
Is question manadatory?

Reply:
Yeah you're right it doesn't make sense to have a wizard without questions.